02_calculator/calculator.rb

```diff
@@ -1 +1,32 @@
 #write your code here
+def add(a, b)
+    return a + b
+end
+
+def subtract(a, b)
+    return a - b
+end
+
+def sum(array)
+    sum = 0
+    array.each do |number|
+        sum += number
+    end
+    return sum
+end
+
+def multiply(a, b)
```

Comment:
This signature will not work when you have a test case with three arguments like:

```
expect(multiply(2, 3, 4)).to eq(24)
```

You'll have to change the signature, not the test.